tests/kubernetes/test_client.py

```diff
@@ -81,3 +84,11 @@ def test_disable_verify_ssl(self):
         else:
             configuration = Configuration()
         assert not configuration.verify_ssl
+
+    @mock.patch("kubernetes.config.incluster_config.InClusterConfigLoader")
+    @conf_vars({("kubernetes", "api_client_retry_configuration"): '{"total": 3, "backoff_factor": 0.5}'})
+    def test_api_client_retry_configuration_correct_values(self, mock_in_cluster_loader):
+        get_kube_client(in_cluster=True)
+        client_configuration = mock_in_cluster_loader().load_and_set.call_args.args[0]
```

Comment:
I think this can solve the problem for python3.7
```suggestion
        client_configuration = mock_in_cluster_loader().load_and_set.call_args[0][0]
```